packages/peregrine/lib/util/shallowMerge.js

```diff
@@ -0,0 +1,3 @@
+const shallowMerge = (...args) => Object.assign({}, ...args);
```

Comment:
My only criticism is that this broke intellisense being able to know the shape of `DEFAULT_OPERATIONS` and providing type hints. That was a huge improvement in this move to peregrine, and I'm hoping it can be retained. Any thoughts on a jsdoc return type that might allow this to work again?